openpgp.d.ts

```diff
@@ -10,6 +10,10 @@
  */
 
 import type { WebStream as GenericWebStream, NodeWebStream as GenericNodeWebStream } from '@openpgp/web-stream-tools';
+import enums from './src/enums';
+import config, { type Config, type PartialConfig } from './src/config';
+
+export { enums, config, PartialConfig };
```

Comment:
`export { enums, config, **Config**, PartialConfig };`

(TODO fix for upstream release)